src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/NotYetReady/If.txt

```diff
@@ -0,0 +1,319 @@
+// ************************************** FIRST CONDITION TRUE CASES **************************************

```

Comment:
If is variadic, so, some tests of the forms

`If(cond1, result1, cond2, result2, cond3, result3, .... resultN, AllFalseResult)`

Reply:
Thanks @lesaltzm I have added few variadic cases after line 90.